### DISCUSSION TITLE
How to setup external clipboard provider?

### DISCUSSION THREAD

**Discussion by dlintw** [helix-editor/helix#9026] [Q&A]
I use [lemonade](https://github.com/lemonade-command/lemonade) to do the clipboard things between Windows and remote ssh linux side.
When I press space+y, it shows.

"clipboard provider tmux failed"

In nvim, it works by default, because nvim support lemonade by default.
I'm wonder how to setup external clipboard provider?

**Comment by the-mikedavis** [comment]
It isn't currently possible. There's an issue for making the clipboard providers configurable: https://github.com/helix-editor/helix/issues/8826
